BuildTools.psm1

```diff
@@ -263,7 +263,7 @@ function Run-TestScriptsOnce([array]$Scripts, [int]$TimeoutSeconds,
         Write-Output "$verb $relativePath..."
         $job = Start-Job -ArgumentList $relativePath, $script.Directory, `
             ('.\"{0}"' -f $script.Name) {
-            $ErrorActionPreference = "Stop"
+            $ErrorActionPreference = "Continue"

```

Comment:
Is this still required, if we're redirecting stderr to stdout anyway? I'm fine with it, but I thought I'd mention it seeming redundant.

Reply:
Yes, it's actually the combination of both that works, redirecting to stdout doesn't work if this is set to "Stop".